css/style.css

```diff
@@ -0,0 +1,83 @@
+html, body, {
```

Comment:
Good attempt at a full page background. Check out this article (https://css-tricks.com/perfect-full-page-background-image/).  That is the current best practice to achieve what you are looking to do here. 🔨